src/main/java/com/github/manolo8/darkbot/backpage/FlashResManager.java

```diff
@@ -72,6 +91,16 @@ public void onTickTask() {
         }
     }
 
+    @Override
+    public void onBackgroundTick() {
+        parseMinimapIds(); // should be lazy-loaded?
```

Comment:
```suggestion
        loadTranslations();
        loadMinimapIds();
```
Yeah we can make those run always, (they each check if they need to refresh or not)